src/sciline/pipeline.py

```diff
@@ -593,7 +611,9 @@ def _build_series(self, tp: Type[Series[KeyType, ValueType]]) -> Graph:
         value_type: Type[ValueType]
         index_name, value_type = get_args(tp)
 
-        subgraph = self.build(value_type, search_param_tables=True)
+        subgraph = self.build(
+            value_type, search_param_tables=True, handler=HandleAsBuildTimeException()
```

Comment:
See above for forwarding handler.

Reply:
See PR message: This is not supported for param tables yet.